croncert-config.yml

```diff
@@ -1738,6 +1738,111 @@ scrapers:
         date_location: "America/Los_Angeles"
         date_language: "en_US"
 
+  ##########
+  # Vevey
+  ##########
+  - name: CafeLitteraire
+    url: https://lecafelitteraire.ch/culture/
+    item: body.culture > div.site > main.site-main > article.is_page.culture.is_parent > div.entry-content > div.wp-block-getwid-section > div.wp-block-getwid-section__wrapper > div.wp-block-getwid-section__inner-wrapper > div.wp-block-getwid-section__content > div.wp-block-getwid-section__inner-content > ul.list.evenements > li
```

Comment:
I removed `renderJs: true` in both locations you added because the outcome of the scraping is the same but it takes significantly less time when not rendering js code.

Reply:
I'm really surprised that it works for The Rocking Chair, but if it works, cool!